terminal_colors/terminal_colors.py

```diff
@@ -94,6 +100,9 @@ def _get_options(args):
         make_option("-z", "--horizontal", action="store_true",
             dest="horizontal", default=False,
             help="Display with horizontal orientation."),
+        make_option("-m", "--force-term", action="store", 
+            metavar="N", choices=list(map(str, TERM_TABLE)),
+            help="Skip termcap color detection and force N colors."),
```

Comment:
OptionParser does not support combining `type="int"` with `choices=`.  I will do a separate PR to switch to `argparse` from `optparse`, because `optparse` is deprecated and argparse supports choice-constrained options that have a type, like `int`.